wct.conf.js

```diff
@@ -34,20 +34,16 @@ module.exports = {
         'Windows 10/microsoftedge@latest-1',
 
         // last 2 Safari major versions (desktop)
-        'macOS 10.13/safari@11.1',
-        'macOS 10.12/safari@10.1',
+        'macOS 10.15/safari@latest',
+        'macOS 10.14/safari@latest',
 
         // last 2 iOS major versions (mobile Safari)
-        'iOS Simulator/iphone@12.0',
-        'iOS Simulator/iphone@11.0',
+        'iOS Simulator/iphone@latest',
+        'iOS Simulator/iphone@latest-1',
 
         // Safari 9 on desktop and mobile
         'OS X 10.11/safari@9.0',
 
-        // The mobile Safari 9 tests are disabled because they fail due to
-        // https://forums.developer.apple.com/thread/36650
-        // 'iOS Simulator/iphone@9.3',
```

Comment:
BTW Removed, it seems not available in SauceLabs anymore.

Reply:
good to legacy code is removed